### DISCUSSION TITLE
Inquiry Regarding Priority Projects for GSoC 2024

### DISCUSSION THREAD

**Discussion by Akhilesh-max** [camicroscope/caMicroscope#672] [General]
Hi, everybody!
I was wondering @birm, if I could be pointed towards the projects that need the most attention or at are priority right now? It would really help me focus my efforts where they're needed the most.

My name is Akhilesh and as an active contributor to open source projects, I have developed proficiency in GitHub Actions, workflows, and the MERN stack. I am eager to apply these skills to contribute to your organization's projects.

Thank you.

**Comment by YashKumarVerma** [comment]
Would recommend you to go through the idea list and see what suits you the most
